python/paddle/distributed/launch/main.py

```diff
@@ -40,9 +40,9 @@ def launch():
 
         - ``--rank``: The rank of the node, can be auto assigned by master. Default ``--rank=-1``.
 
-        - ``--log_level``: The log levl to set for logging.setLevel. Default ``--log_level=INFO``.
+        - ``--log_level``: The log levl to set for logging.setLevel. The rank 0 log will not print in the terminal by default, while you can enable it by adding --log_level=debug. Default ``--log_level=INFO``.
```

Comment:
The log levl -> The log level；
It is better to enumerate the values for log_level and what they mean.